cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -499,10 +516,16 @@ add_install_dir_to_path() {
             # If the script now exists, add the line to source it to the rcfile
             # (This will also create the rcfile if it doesn't exist)
             if [ -f "$_env_script_path" ]; then
-                say_verbose "adding $_robust_line to $_target"
+                local _line
+                if [ "$_shell" = "fish" ]; then
+                    _line="$_pretty_line"
+                else
+                    _line="$_robust_line"
+                fi
```

Comment:
Worth adding a comment here noting that fish has deprecated `. file` and explicitly wants you to use `source file`.

Reply:
Yeah, good call, I'll add a comment